example/svm_mnist/svm_mnist.py

```diff
@@ -41,8 +41,7 @@
 Y = mnist.target[p]
 X_show = mnist.data[p]
 
-# This is just to normalize the input to a value inside [0,1],
```

Comment:
Since we use data processed by ```PCA```, data ```X``` is not the original data which in range [0, 255].
```
>>> np.amax(X)
2383.7941391132126
>>> np.amin(X)
-1525.4982163429079
```